game/assets/jak1/text/game_text_ptbr.gs

```diff
@@ -0,0 +1,346 @@
+(group-name "common")
+(language-id 13) ;; this file is for brazilian portuguese ptbr
+(text-version jak1-v2)
+
+;; -----------------
+;; memcard overrides lol
+
+(#x0138
+  "POR FAVOR NÃO REMOVA OU INSIRA NENHUM PERIFÉRICO, DESLIGUE O SEU SISTEMA OU REMOVA O JOGO"
+  "POR FAVOR NÃO REMOVA OU INSIRA NENHUM PERIFÉRICO, DESLIGUE O SEU SISTEMA OU REMOVA O JOGO"
+  )
```

Comment:
This file is only for 1 language, only specify one text per entry, not two.

Reply:
fixed